client/js/components/pages/dashboard.jsx

```diff
@@ -60,6 +60,16 @@ export default class Dashboard extends React.Component {
             </Card>
           </div>
         </div>
+        <div className='row' style={{marginBottom: '15px'}}>
+          <div className='col-xs-12 col-sm-12'>
```

Comment:
I think it by default take 100% width :P also any `col-xs-` class will apply to all resolutions if it's the only one provided